code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -29,212 +31,264 @@
 	desc = "A small bottle of Chloral Hydrate. Mickey's Favorite!"
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle20"
+	//icon_state = "bottle20"
 
 	New()
 		..()
 		reagents.add_reagent("chloralhydrate", 15)		//Intentionally low since it is so strong. Still enough to knock someone out.
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/antitoxin
 	name = "anti-toxin bottle"
 	desc = "A small bottle of Anti-toxins. Counters poisons, and repairs damage, a wonder drug."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	//icon_state = "bottle17"
 
 	New()
 		..()
 		reagents.add_reagent("anti_toxin", 30)
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/mutagen
 	name = "unstable mutagen bottle"
 	desc = "A small bottle of unstable mutagen. Randomly changes the DNA structure of whoever comes in contact."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle20"
+	//icon_state = "bottle20"
 
 	New()
 		..()
 		reagents.add_reagent("mutagen", 30)
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/ammonia
 	name = "ammonia bottle"
 	desc = "A small bottle."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle20"
+	//icon_state = "bottle20"
 
 	New()
 		..()
 		reagents.add_reagent("ammonia", 30)
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/diethylamine
 	name = "diethylamine bottle"
 	desc = "A small bottle."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	//icon_state = "bottle17"
 
 	New()
 		..()
 		reagents.add_reagent("diethylamine", 30)
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/flu_virion
 	name = "Flu virion culture bottle"
 	desc = "A small bottle. Contains H13N1 flu virion culture in synthblood medium."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle3"
+	//icon_state = "bottle3"
 	New()
 		..()
 		var/datum/disease/F = new /datum/disease/advance/flu(0)
 		var/list/data = list("viruses"= list(F))
 		reagents.add_reagent("blood", 20, data)
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/epiglottis_virion
 	name = "Epiglottis virion culture bottle"
 	desc = "A small bottle. Contains Epiglottis virion culture in synthblood medium."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle3"
+	//icon_state = "bottle3"
 	New()
 		..()
 		var/datum/disease/F = new /datum/disease/advance/voice_change(0)
 		var/list/data = list("viruses"= list(F))
 		reagents.add_reagent("blood", 20, data)
+		update_icon()
 
 /obj/item/weapon/reagent_containers/glass/bottle/liver_enhance_virion
 	name = "Liver enhancement virion culture bottle"
 	desc = "A small bottle. Contains liver enhancement virion culture in synthblood medium."
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle3"
+	//icon_state = "bottle3"
 	New()
 		..()
 		var/datum/disease/F = new /datum/disease/advance/heal(0)
 		var/list/data = list("viruses"= list(F))
 		reagents.add_reagent("blood", 20, data)
+		update_icon()

```

Comment:
It should work without.
Don't stretch the PR into beakers, just don't include them here.
Did you test without?